### ISSUE TITLE
是否支持Kafka 认证

### ISSUE THREAD

**Issue by zhiyuan192shine** [xaecbd/KafkaCenter#14]
请问是否目前Kafka SASL/PLAIN，SASL/SCRAM 或Kerberos 认证。

**Comment by TrumanDu** [comment]
@zhiyuan192shine 目前不支持，未来有支持计划！

**Comment by matthiaspfenninger** [comment]
@TrumanDu Just for clarification, what are the OAuth settings in the application.properties used for (if not for the Kafka authentication)?

**Comment by TrumanDu** [comment]
@matthiaspfenninger yeah, u are right.oauth just for logging into the actual application itself.

**Comment by grbesicb** [comment]
Any updates on this feature?
